src/main/java/games/strategy/internal/persistence/serializable/AaRadarAdvanceProxy.java

```diff
@@ -22,18 +28,16 @@
   public static final ProxyFactory FACTORY = ProxyFactory.newInstance(AARadarAdvance.class, AaRadarAdvanceProxy::new);
 
   private final Map<String, IAttachment> attachments;
-  private final GameData gameData;
 
   public AaRadarAdvanceProxy(final AARadarAdvance aaRadarAdvance) {
     checkNotNull(aaRadarAdvance);
 
     attachments = aaRadarAdvance.getAttachments();
-    gameData = aaRadarAdvance.getData();
   }
 
   @Override
   public Object readResolve() {
-    final AARadarAdvance aaRadarAdvance = new AARadarAdvance(gameData);
+    final AARadarAdvance aaRadarAdvance = new AARadarAdvance(null);
```

Comment:
This is an example of how we now pass `null` for the owning `GameData` instance upon construction of a `GameDataComponent`.  All the other `GameDataComponent` proxies do something similar.